src/test/unit/algorithms/SDRClassifierTest.cpp

```diff
@@ -70,7 +70,7 @@ TEST(SDRClassifierTest, ExampleUsagePredictor)
   // Predict 1 and 2 time steps into the future.
 
   // Make a sequence of 4 random SDRs. Each SDR has 1000 bits and 2% sparsity.
-  vector<SDR> sequence( 4u, vector<UInt>{ 1000u } );
+  vector<SDR> sequence( 4u, SDR({ 1000u }, {}) );
```

Comment:
this is example of a code that used to compile, but shouldn't!

Reply:
This used to be correct.  It still is correct after this change too.
It creates an SDR with dimensions of `[1000]`
Then it randomizes the SDRs, in the for loop that follows this line.

Reply:
true, it is still correct, but I'd say confusing as you silently do the conversion from vec->SDR